blog/2022-11-24-hello-micro-lc-v2.md

```diff
@@ -2,6 +2,42 @@
 title: Welcome micro-lc v2
 description: micro-lc version 2 is out!
 slug: welcome-micro-lc-v2
+authors: paolomartinoli
 ---
 
-_Coming soon..._
+We are proud to announce <micro-lc></micro-lc> **v2.0**! A lot of new features have been implemented and optimized, making the tool more powerful than ever.
+
+Let's take a closer look to what has changed.
+
+## A new live playground
+
+One of the main news is the new [playground](/documentation/playground/) available on the website. Here you can try <micro-lc></micro-lc> live, without the need to install anything: in the box on the right you will directly see the result. You are free to do all the experiments you like and test the full potential of <micro-lc></micro-lc>.
+
+What are you waiting for? Unleash your creativity!
+
+## Using <micro-lc></micro-lc> has never been easier
+
+If you are familiar with <micro-lc></micro-lc> and you have already used it, you will love this new feature. If you are a new user, you will love it too, don't worry.  
+<micro-lc></micro-lc> can now be used through a [CDN](/documentation/docs/getting-started#import-from-cdn): this means that the download will be much lighter and the interface will be easier. This because <micro-lc></micro-lc> and `microlc-element-composer` (which used to be a separated core plugin) are now tightened together.
+
+In addition, the code that makes <micro-lc></micro-lc> run is available as a standalone library that you can download via `npm`.
+
+## New YAML integration
+
+Besides the already supported JSON, we introduced the possibility to write [configuration](/documentation/api/micro-lc-web-component#configuration) in YAML. Since YAML syntax is far more easily understandable for human, this feature will boost you building your microfrontends.
+
+But, if you are a JSON lover, you can still continue using it.
+
+## ... and a lot of other cool stuff
+
+Besides all the above, other new features and improvements have been added:
+
+- The bundle size is very low, so the page loads faster;
```

Comment:
I will put this one in the paragraph where we say that micro-lc is more lighter than before